mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenProjectConfigurator.java

```diff
@@ -6,37 +6,38 @@
 import org.apache.maven.model.Model;
 import org.apache.maven.model.Plugin;
 import org.arquillian.smart.testing.Configuration;
-import org.arquillian.smart.testing.Logger;
 import org.arquillian.smart.testing.mvn.ext.dependencies.DependencyResolver;
 import org.arquillian.smart.testing.mvn.ext.dependencies.ExtensionVersion;
 import org.arquillian.smart.testing.mvn.ext.dependencies.Version;
 import org.codehaus.plexus.util.xml.Xpp3Dom;
 
+import static java.lang.String.format;
 import static org.arquillian.smart.testing.mvn.ext.MavenPropertyResolver.isSkipITs;
 
 class MavenProjectConfigurator {
 
     private static final Version MINIMUM_VERSION = Version.from("2.19.1");
 
-    private static final Logger logger = Logger.getLogger();
+    private final org.codehaus.plexus.logging.Logger logger;
 
     private final Configuration configuration;
 
     private final DependencyResolver dependencyResolver;
 
-    MavenProjectConfigurator(Configuration configuration) {
+    MavenProjectConfigurator(Configuration configuration, org.codehaus.plexus.logging.Logger logger) {
         this.configuration = configuration;
         this.dependencyResolver = new DependencyResolver(configuration);
+        this.logger = logger;
     }
 
     void configureTestRunner(Model model) {
         final List<Plugin> effectiveTestRunnerPluginConfigurations = getEffectivePlugins(model);
 
         if (!effectiveTestRunnerPluginConfigurations.isEmpty()) {
-            logger.debug("Enabling Smart Testing %s for plugin %s in %s module", ExtensionVersion.version().toString(),
+            logger.debug(format("Enabling Smart Testing %s for plugin %s in %s module", ExtensionVersion.version().toString(),
```

Comment:
@bartoszmajsak @MatousJobanek, what I was trying to say was that these debug logs for the plexus logger would only be enabled in the event `-X` flag is set, otherwise it won't.

IMO, this is also logging information related to smart testing extension, and should be available, when just `smart.testing.debug` property is set.

WDYT ?